test/Serilog.Sinks.Console.Tests/Serilog.Sinks.Console.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netcoreapp2.2;netcoreapp3.0;netcoreapp3.1;net452;net462;net472;net48;net5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp2.2;netcoreapp3.0;netcoreapp3.1;net452;net462;net472;net48;net5.0;net6.0;net7.0</TargetFrameworks>
```

Comment:
```suggestion
    <TargetFrameworks>net452;net462;net472;net48;net5.0;net6.0;net7.0</TargetFrameworks>
```

Reply:
@nblumhardt thoughts? Leaner makes sense to me, and this is a test project after all, so targeting standards is not needed.

Reply:
net7 was added here but not in File; we should be consistent one way or the other (I'm fine with adding it in both or neither)